pkg/config/config.go

```diff
@@ -22,7 +22,8 @@ type Config struct {
 	releasesConfig *model.ReleasesConfig
 }
 
-func NewConfig(ctx context.Context, subKey string, refresh time.Duration, channelServerVersion string, urls ...string) (*Config, error) {
+func NewConfig(ctx context.Context, subKey string, refresh time.Duration, channelServerVersion string, urls []string) (*Config, error) {
```

Comment:
This change to this function's signature looks unnecessary.

Reply:
yea, that was from a previous trial, Thank you